src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -262,9 +262,16 @@ local g_tLuaKeywords = {
     ["while"] = true,
 }
 
-local function serializeImpl(t, tTracking, sIndent)
-    local sType = type(t)
+local function serializeImpl(t, tTracking2, sIndent, opts)
+    local sType, tTracking = type(t), {}
     if sType == "table" then
+				if opts.advancedComparing then
```

Comment:
Instead of copying the table, could we just do `tTracking[t] = nil` at the end of this block (and probably move `tTracking[t] = nil` inside the `else` branch).

Reply:
Done